oeps/oep-0018-bp-python-dependencies.rst

```diff
@@ -208,14 +234,18 @@ this file:
   constraint has been imposed.  If there is an issue (either in Jira or an
   upstream issue tracker) for resolving the problem, a link to it should be
   included in the comment.
-* Minimum versions should generally not be included here; ``pip-compile``
+* Minimum versions are generally not necessary for IDAs since ``pip-compile``
```

Comment:
I wonder if this section could have a different intro with sub-bullets. For example:
```
* Minimum constraints are required in some circumstances, and should be avoided in others. See the following rules for details:

  * For IDAs, ...
  * For library packages, ... 
```